server/handle_map_layer_zxy.go

```diff
@@ -183,10 +193,42 @@ func (req HandleMapLayerZXY) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	w.Header().Add("Content-Type", mvt.MimeType)
 	w.Header().Add("Content-Length", fmt.Sprintf("%d", len(pbyte)))
 	w.WriteHeader(http.StatusOK)
-	w.Write(pbyte)
+
+	_, err = w.Write(pbyte)
+	if err != nil {
+		log.Errorf("error writing tile z:%v, x:%v, y:%v - %v", req.z, req.x, req.y, err)
```

Comment:
Hm... Let's see what maintainers think about it, but we might want to log this on another level. It think this could happen e.g. when the client closes the connection or cancels the request and that's extremely common. If you move the map before a tile loads, the request will probably be canceled. This way we may end up spamming the logs a lot.